src/popup/views/RecoverAccount/index.tsx

```diff
@@ -78,20 +92,21 @@ const RecoverAccount = () => {
       validationSchema={RecoverAccountSchema}
     >
       {({ handleChange, isSubmitting, isValid }) => (
-        <Form>
+        <FullHeightForm>
           <Onboarding
             header="Recover wallet from backup phrase"
             icon={icon}
-            goBack={() => history.push({ pathname: "/" })}
+            isMaxHeaderLength
           >
             <>
               <FormRow>
                 <FormTextField
                   as="textarea"
                   name="mnemonicPhrase"
+                  placeholder="Enter your 12 word phrase to restore your wallet"
                   onChange={handleChange}
                 />
-                <FormError name="mnemonicPhrase" />
+                {/* <FormError name="mnemonicPhrase" /> */}
```

Comment:
Textarea had 2 error message boxes. This one wasn't validating anything. Should I remove it?

Reply:
oh yeah - so <FormError name="mnemonicPhrase" /> validates that this field isn't empty before submission (via Formik) while <ApiErrorMessage /> shows an error if there's a problem logging in. We should keep both.
<ApiErrorMessage /> is more of an error for the whole form, so maybe it should be moved closer to the Submit button

Reply:
Added back